addons/io/org.openhab.io.openhabcloud/src/main/java/org/openhab/io/openhabcloud/internal/CloudClient.java

```diff
@@ -634,9 +634,9 @@ public void onHeaders(Response response) {
                     responseJson.put("responseStatusText", "OK");
                     socket.emit("responseHeader", responseJson);
                     logger.debug("Sent headers to request {}", mRequestId);
-                    logger.debug(responseJson.toString());
+                    logger.debug("{}", responseJson.toString());
                 } catch (JSONException e) {
-                    logger.error(e.getMessage());
+                    logger.error("{}", e.getMessage());
```

Comment:
I would log the exception instead, it is an error, I want to be able to debug the stacktrace